<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuSuper</class>
 <widget class="QMainWindow" name="MenuSuper">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Boss</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="creaUsuario">
       <attribute name="title">
        <string>Crear</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="4" column="3">
         <widget class="QLineEdit" name="password"/>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="apLabel">
          <property name="text">
           <string>Sexo</string>
          </property>
          <property name="buddy">
           <cstring>sexo</cstring>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="cedulaLabel">
          <property name="text">
           <string>Cedula</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="contraLabel">
          <property name="text">
           <string>Contraseña</string>
          </property>
          <property name="buddy">
           <cstring>password</cstring>
          </property>
         </widget>
        </item>
        <item row="5" column="3">
         <widget class="QLineEdit" name="nombre"/>
        </item>
        <item row="3" column="3">
         <widget class="QLineEdit" name="usuario"/>
        </item>
        <item row="8" column="1">
         <widget class="QRadioButton" name="radioCedula">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="4">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="nameLabel">
          <property name="text">
           <string>Nombre</string>
          </property>
          <property name="buddy">
           <cstring>nombre</cstring>
          </property>
         </widget>
        </item>
        <item row="8" column="3">
         <widget class="QLineEdit" name="cedula">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="7" column="3">
         <widget class="QLineEdit" name="edad"/>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="telLabel">
          <property name="text">
           <string>Edad</string>
          </property>
          <property name="buddy">
           <cstring>edad</cstring>
          </property>
         </widget>
        </item>
        <item row="9" column="3">
         <widget class="QPushButton" name="pushButton">
          <property name="maximumSize">
           <size>
            <width>85</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Crear</string>
          </property>
         </widget>
        </item>
        <item row="6" column="3">
         <widget class="QLineEdit" name="sexo"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="userLabel">
          <property name="text">
           <string>Usuario</string>
          </property>
          <property name="buddy">
           <cstring>usuario</cstring>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="modificaUsuario">
       <attribute name="title">
        <string>Modificar</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QWidget" name="widget" native="true">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="mUsuario">
             <property name="text">
              <string>Usuario:</string>
             </property>
             <property name="buddy">
              <cstring>mUserLineE</cstring>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLineEdit" name="mUserLineE"/>
           </item>
           <item row="4" column="0">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="2" column="1">
            <widget class="QPushButton" name="mBuscarPB">
             <property name="maximumSize">
              <size>
               <width>85</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Buscar</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QWidget" name="widget_2" native="true">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="userLabel_2">
             <property name="text">
              <string>Usuario</string>
             </property>
             <property name="buddy">
              <cstring>usuario</cstring>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QCheckBox" name="radioUsuario">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="usuario_2">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="contraLabel_2">
             <property name="text">
              <string>Contraseña</string>
             </property>
             <property name="buddy">
              <cstring>password</cstring>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QCheckBox" name="radioPassword">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="password_2">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="nameLabel_2">
             <property name="text">
              <string>Nombre</string>
             </property>
             <property name="buddy">
              <cstring>nombre</cstring>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QCheckBox" name="radioNombre">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QLineEdit" name="nombre_2">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="apLabel_2">
             <property name="text">
              <string>Sexo</string>
             </property>
             <property name="buddy">
              <cstring>sexo</cstring>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QCheckBox" name="radioSexo">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="8" column="1">
            <widget class="QLineEdit" name="sexo_2">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="9" column="0">
            <widget class="QLabel" name="telLabel_2">
             <property name="text">
              <string>Edad</string>
             </property>
             <property name="buddy">
              <cstring>edad</cstring>
             </property>
            </widget>
           </item>
           <item row="10" column="0">
            <widget class="QCheckBox" name="radioEdad">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="10" column="1">
            <widget class="QLineEdit" name="edad_2">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="11" column="0">
            <widget class="QLabel" name="cedulaLabel_2">
             <property name="text">
              <string>Cedula</string>
             </property>
            </widget>
           </item>
           <item row="12" column="0">
            <widget class="QCheckBox" name="radioCedula_2">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="12" column="1">
            <widget class="QLineEdit" name="cedula_2">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="13" column="0">
            <spacer name="verticalSpacer_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="14" column="0">
            <widget class="QPushButton" name="pushButton_2">
             <property name="text">
              <string>Guardar</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="eliminaUsuario">
       <attribute name="title">
        <string>Eliminar</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="1">
         <spacer name="verticalSpacer_5">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="eUser"/>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="eUserPB">
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Eliminar</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <spacer name="verticalSpacer_6">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="0">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="eUserLabel">
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>80</height>
           </size>
          </property>
          <property name="text">
           <string>Usuario</string>
          </property>
          <property name="buddy">
           <cstring>eUser</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="buscaUsuario">
       <attribute name="title">
        <string>Mostrar</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="1" column="0">
         <widget class="QPushButton" name="mostrar">
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Mostrar</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QTableView" name="tableView"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>usuario</tabstop>
  <tabstop>password</tabstop>
  <tabstop>nombre</tabstop>
  <tabstop>sexo</tabstop>
  <tabstop>edad</tabstop>
  <tabstop>radioCedula</tabstop>
  <tabstop>cedula</tabstop>
  <tabstop>mUserLineE</tabstop>
  <tabstop>mBuscarPB</tabstop>
  <tabstop>radioUsuario</tabstop>
  <tabstop>usuario_2</tabstop>
  <tabstop>radioPassword</tabstop>
  <tabstop>password_2</tabstop>
  <tabstop>radioNombre</tabstop>
  <tabstop>nombre_2</tabstop>
  <tabstop>radioSexo</tabstop>
  <tabstop>sexo_2</tabstop>
  <tabstop>radioEdad</tabstop>
  <tabstop>edad_2</tabstop>
  <tabstop>radioCedula_2</tabstop>
  <tabstop>cedula_2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
